### PR TITLE
Document how to customize rules

### PR DESCRIPTION
**Pull request by Majkl578**
* document `phpcs.xml(.dist)?`
* document how to customize ruleset

@jwage Can you please proofread this? Thanks.

### CODE REVIEW

**Comment by Majkl578** [comment]
Also can I easily test this locally (build the docs from my PR)?

**Comment by jwage** [comment]
Yes, go to the projects/coding-standard folder in your doctrine-website clone and checkout your clone & branch. Run `./bin/console build-docs --project=coding-standard` then run `./bin/console build-website`

File: docs/en/reference/customizing.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+.. code-block:: xml
+
+    <rule ref="Doctrine"/>
+
+    <rule ref="Generic.Files.LineLength">
+        <properties>
+            <!-- set soft line length limit to 120 characters -->
+            <property name="lineLimit" value="120"/>
+            <!-- set hard line length limit to 120 characters -->
+            <property name="absoluteLineLimit" value="160"/>
```
Comment (alcaeus) [inline docs/en/reference/customizing.rst]: Mismatch to documentation (120 vs. 160)

File: docs/en/reference/customizing.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    <!-- allow long lines in src/FileWithLongLines.php and in tests/ -->
+    <rule ref="Generic.Files.LineLength">
+        <exclude-pattern>src/FileWithLongLines.php</exclude-pattern>
+        <exclude-pattern>tests/*</exclude-pattern>
+    </rule>
+
+Adding new rules
+----------------
+
+Add new rules is simple, all you need to do is add a new ``<rule>`` block:
```
Comment (greg0ire) [inline docs/en/reference/customizing.rst]: Adding

File: docs/en/reference/index.rst [hunk 3]
```diff
@@ -78,6 +78,45 @@ You might also do automatic fixes using ``phpcbf``:
 
     $ phpcbf --standard=Doctrine /path/to/some/file/to/sniff.php
 
+Project-level ruleset
+=====================
+
+To enable Doctrine Coding Standard for your project, create ``phpcs.xml.dist`` file with the following content:
```
Comment (greg0ire) [inline docs/en/reference/index.rst]: "create a"

File: docs/en/reference/index.rst [hunk 4]
```diff
@@ -78,6 +78,45 @@ You might also do automatic fixes using ``phpcbf``:
...
+        <arg name="colors"/>
+
+        <!-- Ignore warnings, show progress of the run and show sniff names -->
+        <arg value="nps"/>
+
+        <!-- Directories to be checked -->
+        <file>lib</file>
+        <file>tests</file>
+
+        <!-- Include whole Doctrine Coding Standard -->
```
Comment (greg0ire) [inline docs/en/reference/index.rst]: "whole" => "full|the whole"

**Comment by alcaeus** [comment]
🚢

**Pull request merged**